src/Components/Todo/index.jsx

```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from 'react';
 import useForm from '../../hooks/form';
 
 import { v4 as uuid } from 'uuid';
+import List from '../List';
```

Comment:
You're going to feel dumb on this one, but sometimes it just takes another set of eyes. You have imported the `List` component, but are not actually using it; you still have the original `list.map` in the return of this component.

Reply:
OMG, well, you weren't wrong about how I'd feel! smh :P